treatment-service/config/config.go

```diff
@@ -109,12 +117,34 @@ func (c *Config) ListenAddress() string {
 	return fmt.Sprintf(":%d", c.Port)
 }
 
+func (c *Config) Validate() error {
+	validate := validator.New()
+
+	if c.MessageQueueConfig != nil {
```

Comment:
Rather than relying on nil check, does it make sense to have some kind of `SyncMode` to be either using `messageQueue` or `Poller`?

Reply:
I have removed the validator now since both the Queue config and Poller config are required.